samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/test/Function.tests.ps1

```diff
@@ -0,0 +1,14 @@
+Import-Module -Name Pester
```

Comment:
Once the function is updated to return `hello world` then simply update the tests to reflect that so it can capture the body _and_ the statusCode ;)

Here's an example from DotNet core Tests: https://github.com/awslabs/aws-sam-cli/blob/0622b9bb9b601a6c8c1e4054c2a0c93c2e70c96b/samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/%7B%7Bcookiecutter.project_name%7D%7D/test/HelloWorld.Test/FunctionTest.cs